app/assets/javascripts/discourse/templates/preferences/users.hbs

```diff
@@ -0,0 +1,29 @@
+{{#if siteSettings.ignore_user_enabled}}
```

Comment:
and then only do this check for the ignored part

Reply:
Makes sense. I was hesitated to make the UI change (it might seem to be dramatic) but I guess it is okay 👍